cloudSupport/src/main/scala/cromwell/cloudsupport/aws/s3/S3Storage.scala

```diff
@@ -42,11 +43,13 @@ object S3Storage {
     val dualstackEnabled = ConfigFactory.load().as[Option[Boolean]]("s3.dual-stack").getOrElse(false)
     val pathStyleAccessEnabled = ConfigFactory.load().as[Option[Boolean]]("s3.path-style-access").getOrElse(false)
 
-    S3Configuration.builder
+    @nowarn("msg=method dualstackEnabled in trait Builder is deprecated")
```

Comment:
Is there a different way to enable dualstack, whatever that is? Or is it a dead feature that we should remove support for?

Reply:
There is an alternative way of enabling dualstack per the more [detailed suggestions](https://github.com/aws/aws-sdk-java-v2/blob/4656690aed8effe36fcf31cec695778992ee8228/services/s3/src/main/java/software/amazon/awssdk/services/s3/S3Configuration.java#L229-L231) in the deprecation annotation, but I was not easily able to restructure the existing code to use that. Given the priorities around this dependency this seemed like a reasonable approach.